datafusion/sql/tests/sql_integration.rs

```diff
@@ -2176,148 +2176,6 @@ fn union_all() {
     quick_test(sql, expected);
 }
 
-#[test]
-fn union_with_different_column_names() {
```

Comment:
Converting these union tests to SLT is easier than moving them to `TypeCoercion`.